smoketest/src/helper.rs

```diff
@@ -184,7 +184,7 @@ pub async fn send_sudo_xcm_transact(
 	penpal_client: &Box<OnlineClient<PenpalConfig>>,
 	message: Box<VersionedXcm>,
 ) -> Result<ExtrinsicEvents<PenpalConfig>, Box<dyn std::error::Error>> {
-	let dest = Box::new(VersionedMultiLocation::V3(MultiLocation {
+	let dest = Box::new(VersionedLocation::V3(MultiLocation {
```

Comment:
Are we not using XCM V4 now, on BridgeHub? 

Thanks for updating these, by the way! I forgot about updating them.

Reply:
Yes, it's v4 now. But seems it's compatible with v3 and I tested the smoke tests still works.

I would prefer to update the smoke tests in a separate PR later.